src/screens/Episode/helpers/mapEpisodeToPlayerEpisode.ts

```diff
@@ -6,7 +6,7 @@ import {
 
 const mapEpisodeToPlayerEpisode = (
   episode: Episode,
-  author: string
+  author = 'Undefined'
```

Comment:
What is this for?

Reply:
Thanks to navigation props we don't obtain the `author` prop always. Therefore we need default value for it.
![image](https://user-images.githubusercontent.com/60543500/132498148-09b46064-5970-4a7d-8666-8d87facea55b.png)

Reply:
> What is this for?

Thanks to navigation props we don't obtain the `author` prop always. Therefore we need default value for it.

![image](https://user-images.githubusercontent.com/60543500/132498148-09b46064-5970-4a7d-8666-8d87facea55b.png)